components/pickerbutton/package.json

```diff
@@ -52,5 +52,6 @@
 	],
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"componentGuidelinesName": ""
```

Comment:
On the docs site, the [picker button guidelines link](https://opensource.adobe.com/spectrum-css/pickerbutton.html) actually points to the [picker guidelines](https://spectrum.adobe.com/page/picker/). 

Do we want to replicate that link for picker button? Currently, picker button isn't rendering a guidelines card.